addons/core/functions/fnc_haveSWRadio.sqf

```diff
@@ -31,6 +31,13 @@ if (isNil "TFAR_currentUnit" || {isNull (TFAR_currentUnit)}) exitWith {false};
     true;
 } count (assignedItems TFAR_currentUnit);
 
+if (player != TFAR_currentUnit) then {
```

Comment:
Yes it is. The Zeus player could not have a radio but is remote controlling a unit that does.